server/analysis/tests/test_gpr.py

```diff
@@ -96,11 +128,11 @@ def test_gprnp_sigmas(self):
 
 
 # test Gradient Descent in GPFlow model
-class TestGPRGP(unittest.TestCase):
+class TestGPFGD(unittest.TestCase):
```

Comment:
I think it is better to rename it to TestGPRGPFGD to be consistent. Or we can rename the first class to TestGPF.